index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/css/global.css">
+  <link rel="stylesheet" href="/css/home.css">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <title>판다마켓</title>
+</head>
+<body>
+  <header>
+    <div class="header-container max-container ">
+      <a href="/">
+        <img src="/images/logo.png" alt="판다 로고" />
+      </a>
+      <button class="btn-login" onclick="location.href='/login.html'">로그인</button>
```

Comment:
P3:
페이지 이동의 경우 a tag 를 추천드립니다.

```suggestion
      <a class="btn-login"  href="./login">로그인</button>
```